Add more descriptive error message

- now test case inputs support value and sensitive tags
- input: {a:b} worked before, now throws exception
- maybe refine the current error message to be more specific:
```
12:48:02 [ERROR] Exception: There was a problem parsing the YAML source: get_values.input
s.
Artifact has unrecognized tag {laptop}. Please take a look at the supported features per
versions link
```

--- a/cloudslang-content-verifier/src/main/java/io/cloudslang/lang/tools/build/tester/parse/TestCasesYamlParser.java
+++ b/cloudslang-content-verifier/src/main/java/io/cloudslang/lang/tools/build/tester/parse/TestCasesYamlParser.java
@@ -73,7 +73,8 @@ public class TestCasesYamlParser {
                 }
             });
         } catch (Throwable e) {
-            throw new RuntimeException("There was a problem parsing the YAML source: " + source.getFileName() + ".\n" + e.getMessage(), e);
+            throw new RuntimeException("There was a problem parsing the YAML source: " + source.getFileName() +
+                    source.getFileExtension().getValue() + ".\n" + e.getMessage(), e);
         }
     }
 
@@ -123,7 +124,8 @@ public class TestCasesYamlParser {
             if (!knownModifierKeys.contains(modifierKey)) {
                 throw new RuntimeException(
                         "Artifact has unrecognized tag {" + modifierKey + "}" +
-                                ". Please take a look at the supported features per versions link");
+                                ". Supported tags are {" + SlangTextualKeys.VALUE_KEY + "} and {" + SlangTextualKeys.SENSITIVE_KEY +"}. " +
+                                "Please take a look at the supported features per versions link");
             }
         }
     }

--- a/cloudslang-content-verifier/src/main/java/io/cloudslang/lang/tools/build/tester/parse/TestCasesYamlParser.java
+++ b/cloudslang-content-verifier/src/main/java/io/cloudslang/lang/tools/build/tester/parse/TestCasesYamlParser.java
@@ -125,7 +125,7 @@ public class TestCasesYamlParser {
                 throw new RuntimeException(
                         "Artifact has unrecognized tag {" + modifierKey + "}" +
                                 ". Supported tags are {" + SlangTextualKeys.VALUE_KEY + "} and {" + SlangTextualKeys.SENSITIVE_KEY +"}. " +
-                                "Please take a look at the supported features per versions link");
+                                "Please take a look at the supported features per versions link.");
             }
         }
     }

--- a/cloudslang-content-verifier/src/test/java/io/cloudslang/lang/tools/build/tester/parser/TestCasesYamlParserTest.java
+++ b/cloudslang-content-verifier/src/test/java/io/cloudslang/lang/tools/build/tester/parser/TestCasesYamlParserTest.java
@@ -83,6 +83,16 @@ public class TestCasesYamlParserTest {
     }
 
     @Test
+    public void testSimpleTestCasesParsingInvalidKey() throws URISyntaxException {
+        String filePath = "/test/base/test_print_text-unrecognized_tag.inputs.yaml";
+        URI fileUri = getClass().getResource(filePath).toURI();
+        exception.expect(RuntimeException.class);
+        exception.expectMessage("Artifact has unrecognized tag {invalid_key}. Supported tags are {value} and {sensitive}. " +
+                "Please take a look at the supported features per versions link.");
+        parser.parseTestCases(SlangSource.fromFile(fileUri));
+    }
+
+    @Test
     public void testCaseFileParsingForNonTestCasesFile() throws Exception{
         String filePath = "/content/base/properties.prop.sl";
         URI fileUri = getClass().getResource(filePath).toURI();

--- a/cloudslang-content-verifier/src/test/resources/test/base/test_print_text-unrecognized_tag.inputs.yaml
+++ b/cloudslang-content-verifier/src/test/resources/test/base/test_print_text-unrecognized_tag.inputs.yaml
@@ -1,0 +1,21 @@
+testPrintFinishesWithSuccess:
+  inputs:
+    - text: text to print
+    - password: {invalid_key: password1, sensitive: true}
+  description: Tests that print_text operation finishes with SUCCESS
+  testFlowPath: base.test_print_text-SUCCESS
+  result: SUCCESS
+
+testPrintFinishesWithFailure:
+  inputs:
+    - text: orit
+  description: Tests that print_text operation finishes with FAILURE
+  testFlowPath: base.test_print_text-SUCCESS
+  result: FAILURE
+
+
+testPrintFinishesWithSuccess2:
+  inputs:
+    - text: text to print
+  description: Tests that print_text operation finishes with SUCCESS 2
+  testFlowPath: base.test_print_text-SUCCESS

